### ISSUE TITLE
Post-deliverables

### ISSUE THREAD

**Issue by ericadamski** [evanthefive/simple-chat-app#4]
You guys havn't complete [post-deliverables](https://github.com/ericadamski/simple-chat-app/wiki/Post-Deliverable-Checklist) or this weeks [pre-deliverables](https://github.com/ericadamski/simple-chat-app/wiki/Pre-Deliverable-Checklist), you can have until sunday to complete these, slack me if you have any questions @delg0030 @evanthefive

**Comment by delg0030** [comment]
oooops. Thanks Eric! We're working on it now :)

**Comment by evanthefive** [comment]
@ericadamski @delg0030 

So far Sweet & Salty has solidified a wireframe to continue forward with as well as the design decisions we would like to implement. We are now looking to complete our interactive wireframes. We had created the paper wireframe for the past deliverable… As for positives; Sweet & Salty can proudly say they have learned how to communicate and how to share ideas in a more proficient way. Our group also has a strong understanding of what we would like our final product to look like. We found working within GitHub and Slack to be very confusing and frustrating. Lemme know what you think and where we be heading Eric, sorry for the late submission!
